### DISCUSSION TITLE
b-jet contamination in the VBF category

### DISCUSSION THREAD

**Discussion by ram1123** [ram1123/2l2Q_limitSettingTool#45] [Comments_HIG_23_001] [answered]
b-jet contamination in the VBF category

**Comment by jialin-guo1** [comment]
answer Link here **slide 14** https://github.com/users/ram1123/projects/2/views/1?pane=issue&itemId=24103306

**Reply by ram1123** [reply, marked as answer]
I copied the numbers from there to here.

# Resolved category

|    |        **vbf500**     |     **vbf1000**     |     **vbf1500**     |    **vbf2500**     |   **vbf**     |   **Bkg**   |
| --- | --- | --- | --- | --- | --- | --- |
|   **n\_vbf\_all**      |   1050.919386   |   86.415901   |   13.356579   |   2.376387   |   4761.332242   |   377.988163   |
|   **n\_vbf\_btag**     |   141.582093   |   17.037556   |   2.418644   |   0.459665   |   720.252417   |   7.422618   |
|   **fraction**       |   0.134722   |   0.197158   |   0.181083   |   0.193430   |   0.151271   |   0.019637   |


# Merged category

|    |        **vbf500**     |     **vbf1000**     |     **vbf1500**     |    **vbf2500**     |   **vbf**     |   **Bkg**   |
| --- | --- | --- | --- | --- | --- | --- |
|   **n\_vbf\_all**      |   1481.956303   |   1034.509960   |   391.081195   |   175.602182   |   15074.913335   |   59.106170   |
|   **n\_vbf\_btag**     |   384.649522   |   275.311235   |   95.604279   |   39.777467   |   3989.038098   |   10.498964   |
|   **fraction**       |   0.259555   |   0.266127   |   0.244461   |   0.226520   |   0.264614   |   0.177629   |

**Reply by ram1123** [reply]
attaching the slide mentioned in the thread:

[program_for_pre_approval.pdf](https://github.com/user-attachments/files/22563357/program_for_pre_approval.pdf)
[program_for_pre_approval.key.zip](https://github.com/user-attachments/files/22563360/program_for_pre_approval.key.zip)
